src/datasets/search.md

```diff
@@ -21,7 +21,7 @@ this is likely your issue.
 To get access permissions, file a bug using the [search permissions template]
 
 Once you have proper permissions,
-you'll have access to a new source in re:dash called `Presto Search`.
+you'll have access to a new source in re:dash called FIXME `Presto Search`.
```

Comment:
I just filed #367 to handle updating the search dataset docs, might be best to just handle this there.

Reply:
I fixed these FIXME's here, but it's still a good idea to review and update this article specifically.